static/src/js/pages/DraftPage/DraftPage.jsx

```diff
@@ -169,7 +167,7 @@ const DraftPageHeader = () => {
   return (
     <>
       <PageHeader
-        title={data?.draft?.previewMetadata?.pageTitle || '<Blank Name>'}
+        title={previewMetadata?.pageTitle || '<Blank Name>'}
```

Comment:
Do defaults during destructure not work here? Assuming this comes back as null?

Reply:
Right, pageTitle is null so defaults do not work